airflow/config_templates/config.yml

```diff
@@ -64,7 +64,9 @@
       description: |
         The SqlAlchemy connection string to the metadata database.
         SqlAlchemy supports many different database engine, more information
-        their website
+        their website. You will need a user and password provided in the database
```

Comment:
It seems to me that it is not worth describing individual drivers here, but we can add a link to the documentation describing ttis in detail.
http://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#database-uri

Reply:
Thanks for the review @mik-laj - just changed it in the recent commit. Apologies for the delay !